internal/app/machined/pkg/controllers/kubespan/nftables.go

```diff
@@ -324,6 +363,130 @@ func matchIPSet(set *nftables.Set, mark, mask uint32, family nftables.TableFamil
 	}
 }
 
+func matchIPSetMSS(set *nftables.Set, mtu uint32, family nftables.TableFamily) []expr.Any {
+	var (
+		offset uint32 = 16
+		length uint32 = 4
+		ifmtu  uint16
+		mss    uint16
+	)
+
+	switch {
+	case mtu == 0:
+		ifmtu = 1500
+	case mtu < 576: // Minimum IPv4 mtu size
+		ifmtu = 576 - 40
```

Comment:
bug here..
```
ifmtu = 576
```

Reply:
looks like with Linux 6.1 minimum MTU will be 1280 https://lore.kernel.org/wireguard/20190321033638.1ff82682@natsu/t/